Paccy10/olympus-api
# Feature(bookings): create a booking

**What does this PR do?**

- Property booking

**Description of Task to be completed?**

- An authenticated user can book any published available property

**How should this be manually tested?**

- Checkout to branch `ft-create-booking`
- Run the app with `flask run`
- Send a `POST` request to `/properties/{property_id}/book` with the required payload  and replace `property_id` with the property ID
- A new booking record should be added to the database

**Any background context you want to provide?**

- N/A

**What are the relevant pivotal tracker stories?**

- N/A

**Screenshots (if appropriate)**

- N/A

**Questions:**

- N/A
